spec/features/menu_bar_spec.rb

```diff
@@ -50,11 +50,11 @@
     let(:admin) { create(:user, permissions: {student: 'private/*', admin: 'private/*'}) }
     let(:owner) { create(:user, permissions: {student: 'private/*', owner: 'private/*'}) }
 
-    scenario 'visitor should only see their account' do
+    scenario 'visitor should only see sing out' do
```

Comment:
Could we add some more feature tests for the several scenarios of access modes and organization's statuses, please?